components/Chips/examples/ChipsActionExampleViewController.m

```diff
@@ -27,23 +27,16 @@ @implementation ChipsActionExampleViewController {
 - (id)init {
   self = [super init];
   if (self) {
-    self.colorScheme =
+    MDCContainerScheme *scheme = [[MDCContainerScheme alloc] init];
+    scheme.colorScheme =
         [[MDCSemanticColorScheme alloc] initWithDefaults:MDCColorSchemeDefaultsMaterial201804];
-    self.shapeScheme = [[MDCShapeScheme alloc] init];
-    self.typographyScheme =
-        [[MDCTypographyScheme alloc] initWithDefaults:MDCTypographySchemeDefaultsMaterial201804];
+    scheme.typographyScheme = [[MDCTypographyScheme alloc] init];
+    scheme.shapeScheme = [[MDCShapeScheme alloc] init];
```

Comment:
We should be able to delete these two assignments here and throughout the examples. Container schemes rely on the default behavior to apply default theming.

Reply:
@jverkoey I agree your request to remove the assignments in this case - the assignments are effectively no-ops because we're using theming extensions and nothing else.  However, container schemes in general do not rely on any behavior to "apply default theming." That's the behavior of *theming extensions* or themers (which have undefined behavior for `nil` values).

Reply:
Yep, I just wanted to make sure any readers didn't misunderstand what how container schemes are expected to work (they don't have any behavior).